.github/workflows/pr.yaml

```diff
@@ -96,7 +99,7 @@ jobs:
         run: |
           # deploy example-apiserver using a passed KONK reference
           make deploy-apiserver KIND_NAME="chart-testing" KONK_NAME="runner-konk"
-          POD_NAME=$(kubectl get pod -o name -l app.kubernetes.io/name=konk-service )
+          POD_NAME=$(kubectl get pod -o name -l app.kubernetes.io/name=konk-service,app.kubernetes.io/component=apiservice )
```

Comment:
If I understand this correctly, the apiserver gets installed into `default`. So moving konk into a different namespace tests cross-namespace support

Reply:
Yes, you understood correctly. That's why the PR moves the konk into `konk` namespace.